package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "eslint-config-deloitte",
+  "private": true,
+  "devDependencies": {
+    "lerna": "^2.8.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/DeloitteDigitalAPAC/eslint-config-deloitte.git"
+  },
+  "author": "Deloitte Digital AU",
+  "license": "BSD-3-Clause",
+  "scripts": {
+    "bootstrap": "./node_modules/.bin/lerna bootstrap",
```

Comment:
`npm run` adds `./node_modules/.bin/` to the PATH, so we could simplify this line to:
`"bootstrap": "lerna bootstrap",`

Reply:
I didnt know NPM did that.. cool. I always assumed that required a global install. I'll update the PR.